pages/index.js

```diff
@@ -23,6 +23,7 @@ const HomeScreen = ({ user, setUser }) => {
   };
   const jumpToStep = (step) => {
     const isFirstStep = active === 0;
+    const hasEquipment = user.equipment && user.equipment.length > 0;
```

Comment:
This const is already defined two lines below - it's using the useState variable, which is initialized with user.equipment - so it's kinda same

Reply:
yea... quick undefined navigator fix lol sry. Fixed

Reply:
awesome!

Reply:
btw I just added some issues to the repository if you're looking for some tasks :) Also feel free to come up with stuff you think would be cool

ps. I won't have time this weekend to review anything because I'm on a Diablo 4 LAN 😅